src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.TimeDate.UnitTests/ImageTests.cs

```diff
@@ -60,6 +60,7 @@ public void Setup()
         [DataRow("iso zone", "ISO 8601 with time zone - ", "Images\\timeDate.dark.png")]
         [DataRow("iso utc zone", "ISO 8601 UTC with time zone - ", "Images\\timeDate.dark.png")]
         [DataRow("rfc", "RFC1123 -", "Images\\timeDate.dark.png")]
+        [DataRow("compatible", "Date and time in filename-compatible format", "Images\\timeDate.dark.png")]
```

Comment:
Why don't we use icons again?
![image](https://user-images.githubusercontent.com/65828559/227782422-8421bf2b-be0b-4629-9af1-024db62cba38.png)

Reply:
Because this is the way it works since ever in every plugin. We can't use glyphs as result symbol because it is not supported. And with images we are much more flexible.

Reply:
You just saved me at least an hour of going through the code and trying to figure out how to switch to icons (-:

Just in general. Is something more needed from my side to have the PR eventually merged? I know that I did not check some checkboxes on the PR checklist, but I am not sure how to handle those, so some assistance there would be appreciated. To be clear I don't expect the PR to be merged overnight or anytime soon for that matter - just am trying to make sure, that it is not some inaction on my part blocking the possibility of the merge in the future.

Reply:
@picazzso
The check boxes are okay: The docs need no update and there are no new files.
I approved the PR today. You can request a review from @jaimecbernardo or @stefansjfw as they can approve the PR to unblock it and they can merge it.